discovery-infra/install_cluster.py

```diff
@@ -11,6 +11,15 @@
 from logger import log
 
 
+def _get_tf_folder():
+    cluster_name = args.cluster_name or consts.CLUSTER_PREFIX
+    cluster_name += f'-{args.namespace}'
+    return os.path.join(
+        consts.TF_FOLDER,
+        f'{cluster_name}__{args.namespace}'
```

Comment:
missing something. One place u have cluster_name:namespace and here {cluster_name}__{args.namespace}
?

Reply:
I started with '__' as you see in this commit but I wanted a separator that cant be passed as part of the namespace or cluster name.